src/main/jbake/content/blog/2015/11/19/hawkular-1.0.0.Alpha7-released.adoc

```diff
@@ -0,0 +1,54 @@
+= Yet another release of Hawkular, the seventh one !
+Thomas Heute
+2015-11-19
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, hawkular, release
+
+Hawkular, the monitoring platform, is happy to announce the release of its seventh milestone 1.0.0.Alpha7.
+
+=== Alert Center
+
+More progress on the alert center, with the option to list and search all the alerts definitions (the rules) and modify those.
```

Comment:
"with the option to list and search all alerts definitions (the rules) and modify those"
What about:
"with the option to list, search and edit triggers (the rules of the alerts definitions)"